deploy/templates/notificationtemplates/stonesoup/userprovisioned/subject.txt

```diff
@@ -0,0 +1 @@
+Notice: Your Stone soup account is provisioned.
```

Comment:
See https://docs.google.com/document/d/11P8Gj8Q9gindbB9wmbFuIKv53SQL3b99vHUy0jemPfw/edit

Reply:
done in [3df1eab](https://github.com/codeready-toolchain/host-operator/pull/754/commits/3df1eab38ec6e59824567e8d6c9d39bd6c43a3ff)